civihr_default_theme/template.php

```diff
@@ -167,9 +167,12 @@ function civihr_default_theme_form_element_label($variables) {
  * @return array
  */
 function bootstrapable_fields($fields_structure) {
-  return array_filter($fields_structure, function($value, $key) {
-    return substr($key, 0, 1) <> '#' && !in_array($value['#type'], ['hidden']);
-  }, ARRAY_FILTER_USE_BOTH);
+  foreach($fields_structure as $key => $value)  {
+    if (!(substr($key, 0, 1) <> '#' && !in_array($value['#type'], ['hidden'])))  {
+      unset($fields_structure[$key]);
```

Comment:
the value is not passed by reference to this function so it doesn't matter in this case ... but I will do it just case there is some hidden drupal magic that I am not aware of .